### DISCUSSION TITLE
Hide notes?

### DISCUSSION THREAD

**Discussion by NorPhi** [TriliumNext/Trilium#1800] [Q&A]
I am a little bit annoyed by widgets being part of the note tree and cluttering up the similar note list. Is there a way to hide them?

Btw. any reason for those things being part of the note tree? It seems odd to me to not have a designated place for plugins.

**Comment by zadam** [comment]
You can put `archived` label on them and then hide them in the tree:

![image](https://user-images.githubusercontent.com/617641/112765366-3306f280-900d-11eb-8405-673b90836ff5.png)

Plugins/widgets being notes makes a lot of things easier. If they are not in the tree for example you would need some special UI to access them. Categorizing them in the tree can be sometimes truly useful too

**Reply by NorPhi** [reply]
This will still make them show up as similar notes. I guess this makes sense for archived notes. Is there a label to suppress notes in that list?

Maybe adding a note type for plugins and themes would be a nice solution. They could be set up to be hidden by default not interfere with the normal note taking unless explicitly labelled as part of the tree. This would also open up the possibility of adding a plugins tab to the options if that is ever needed.

**Reply by zadam** [reply]
Archived notes are penalized when computing "similarity" (although they can still show up there if there are enough other similarities).

BTW are you talking about the widgets provided in the demo content or your own widgets? If you don't like / use those demo widgets you can simply delete them and they won't bother you ...

**Reply by NorPhi** [reply]
I like them, although I was a little confused why some of those are implemented as widgets, I probably would have used note type for things like tasks and journal.
